src/Toolkit/Toolkit.WPF/UI/Controls/BasemapGallery/BasemapGallery.Theme.xaml

```diff
@@ -193,13 +217,44 @@
                                     Stretch="UniformToFill" />
                             </Grid>
                         </Border>
-                        <TextBlock
-                            x:Name="ItemTextBlock"
+                        <Grid
                             Grid.Column="1"
-                            Margin="0,6,6,6"
-                            VerticalAlignment="Center"
-                            Text="{Binding Name, Mode=OneWay}"
-                            TextWrapping="Wrap" />
+                            Margin="0,6,6,6">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="Auto" />
+                            </Grid.RowDefinitions>
+                            <TextBlock
+                                x:Name="ItemTextBlock"
+                                Grid.Row="0"
+                                VerticalAlignment="Center"
+                                Text="{Binding Name, Mode=OneWay}"
+                                TextWrapping="Wrap" />
+                            <Border
+                                Background="LightGray"
+                                CornerRadius="7"
+                                Grid.Row="1"
+                                Padding="4,0"
+                                HorizontalAlignment="Left"
+                                VerticalAlignment="Bottom">
+                                <Border.Style>
+                                    <Style TargetType="Border">
+                                        <Style.Triggers>
+                                            <DataTrigger Binding="{Binding Is3D}" Value="false">
+                                                <Setter Property="Visibility" Value="Hidden" />
```

Comment:
Use collapsed instead of hidden. It doesn't need to take up space when not rendering